android/quest/src/main/assets/configurations/app/application_workflow.json

```diff
@@ -44,6 +44,12 @@
         "resource": "PatientRegisterRowViewConfiguration",
         "classification": "patient_register_row",
         "description": "Configurations for Patient Row"
+      },
+      {
+        "workflowPoint": "test_result_detail_view",
+        "resource": "DetailViewConfiguration",
+        "classification": "test_result_detail_view",
+        "description": "Configurations for Test Results Details"
       }
```

Comment:
I think the above config also needs to add in the quest mapping. cc @ellykits

Reply:
done!